.github/workflows/cesar.yaml

```diff
@@ -0,0 +1,121 @@
+name: CESARt
+on:
+  pull_request:
+    types: [ labeled ]
+
+
+env:
+  ENGINE_VERSION: ${{ vars.ENGINE_VERSION }}
+  PLATFORM: "LINUX_X64"
+  ENGINE: "2ms"
+  CES_ENVIROMENT: "prod"
+  LOG_FILE: ${{ github.workspace }}/log.log
+  REMOVE_HISTORY: "true"
+
+jobs:
+  cesar:
+    if: (github.event.label.name == 'cesar' && github.event.pull_request.mergeable == true)
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+        with:
+          ref: refs/pull/${{ github.event.pull_request.number }}/merge
+          path: 2ms
+
+      - uses: actions/setup-go@0c52d547c9bc32b1aa3301fd7a9cb496313a4491 # v5.0.0
+        with:
+          go-version: "^1.22"
```

Comment:
I would use the go version that 2ms actually builds upon, like this:
`go-version-file: 2ms/go.mod`